cds/modules/deposit/api.py

```diff
@@ -75,3 +78,117 @@ def _process_files(self, record_id, data):
     def multipart_files(self):
         """Get all multipart files."""
         return MultipartObject.query_by_bucket(self.files.bucket)
+
+
+def video_resolver(ids):
```

Comment:
I preferred to leave the resolver more generic to be able to reuse it.
If you need to convert url -> videos you can do:
```python
video_resolver([record_unbuild_url(video_ref) for video_ref in refs])
```